src/components/Home/VideoItem.jsx

```diff
@@ -15,9 +14,13 @@ const VideoText = styled.p`
 `;
 
 export const VideoItem = (props) => {
-  // console.log(props);
+  const { setSelectedVideo } = props;
+  const handleVideoClick = (video) => {
+    setSelectedVideo(video);
+    // console.log(video);

```

Comment:
Remember to remove the console logs when you are sending a PR.

Reply:
Sorry, got it!

Reply:
And for testing, had a session with another member of the bootcamp and finally figured out how to get the coverage, will work on testing these days 😁